%LaTeX template : http://systbio.org/files/SB_LaTeX_Template_txt_extension.txt
%Author instructions: http://www.oxfordjournals.org/our_journals/sysbio/for_authors/ms_preparation.html


\documentclass[12pt,letterpaper]{article}

%Packages
\usepackage{fixltx2e}
\usepackage{textcomp}
\usepackage{fullpage}
\usepackage{natbib}
\usepackage{float}
\usepackage{latexsym}
\usepackage{hyperref}
\usepackage{url}
\usepackage{hyperref}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
\usepackage{amsthm}
\usepackage{amstext}
\usepackage{enumerate}
\usepackage[osf]{mathpazo}
\pagenumbering{arabic}

%Pagination style and stuff
\linespread{1.66}
\raggedright
\setlength{\parindent}{0.5in}
\setcounter{secnumdepth}{0} 
\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}
\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}
\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}
\renewcommand{\tableofcontents}{}
\bibpunct{(}{)}{;}{a}{}{,}

%---------------------------------------------
%
%       START
%
%---------------------------------------------

\begin{document}
%\modulolinenumbers[1]     % Line numbering on every line

%Running head
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: Missing data and topology in total evidence approach

\bigskip
\medskip
\begin{center}

\noindent{\Large \bf Effect of missing data on topological inference using a total evidence approach}
% TG: Or "Effect of missing data and inference method on topology when using a total evidence approach"
% TG: Or any other suggestions?

\bigskip

%Authors
\noindent {\normalsize \sc Thomas Guillerme$^1$$^,$$^2$, and Natalie Cooper$^1$$^,$$^2$}\\
\noindent {\small \it 
$^1$School of Natural Sciences, Trinity College Dublin, Dublin 2, Ireland;\\
$^2$Trinity Centre for Biodiversity Research, Trinity College Dublin, Dublin 2, Ireland;}\\
\end{center}
\medskip
\noindent{\bf Corresponding author:} Thomas Guillerme, School of Natural Sciences, Trinity College Dublin, Dublin 2, Ireland; E-mail: guillert@tcd.ie; Fax: +353 1 6778094; Tel: +353 1 896 2571.\\
\vspace{1in}

%---------------------------------------------
%
%       ABSTRACT
%
%---------------------------------------------

\newpage
\begin{abstract}
Living species represent a marginal part of all species that have ever lived. Ignoring fossil taxa may lead to misinterpretation of macroevolutionary patterns and processes such as trends in species richness, biogeographical history or paleoecology. This fact has led to an increasing consensus among scientists that both living and fossil taxa must be included in macroevolutionary studies. One approach called Total Evidence, uses molecular data from living taxa and morphological data from both living and fossil taxa to infer phylogenies with both living and fossil taxa at the tips. Although the Total Evidence approach seems very promising, it requires a lot of data and is therefore likely to suffer from missing data issues which may affect its ability to infer correct phylogenies.

In this study we assess the effect of missing data on tree topologies inferred from Total Evidence matrices. Using simulations we investigate three major factors that directly affect the completeness of the morphological part of the matrix: (1) the proportion of living taxa with no morphological data, (2) the amount of missing data in the fossil record, and (3) the overall number of morphological characters in the matrix. 

We find that, when using a clade conservative metric such as Robinson-Foulds distance, Bayesian consensus trees recovers the right topology better than Maximum Likelihood or than a subset of Bayesian posterior tree distributions and this regardless of the amount of missing data (minimal tree similarity of 0.69 in the worst scenario). Regarding the data, good topological recovery is not related to the amount of missing data \textit{per se} but to the amount of data overlap. The two factors that influences the most this data overlap are the parameters 1 (proportion of coded living taxa) and 3 (number of morphological characters).

%Probably needs to be deeeply rewritten, as said last time to Kev quoting Stuart Pim, people only read the abstract.
% NC: Yep, for now I'll leave this

\end{abstract}

\noindent (Keywords: missing data, Total Evidence, Bayesian, Maximum Likelihood, topology)\\

\vspace{1.5in}

\newpage 

%---------------------------------------------
%
%       INTRODUCTION
%
%---------------------------------------------

\section{Introduction}

Although most species that have ever lived are now extinct \citep{novacek1992ext,raup1993extinction}, the majority of macroevolutionary studies focus solely on living species \citep[e.g.][]{meredithimpacts2011,jetzthe2012}. Ignoring fossil taxa may lead to misinterpretation of macroevolutionary patterns and processes such as the timing of diversification events \citep[e.g.][]{pyrondivergence2011}, relationships among lineages \citep[e.g.][]{manosphylogeny2007} or niche occupancy \citep[e.g.][]{pearmanniche2008}. This has led to increasing consensus among evolutionary biologists that fossil taxa should be included in macroevolutionary studies \citep{jacksonwhat2006,quentaldiversity2010,dietlconservation2011,slaterunifying2013,fritzdiversity2013}. However, to do this we need to be able to place living and fossil taxa into the same phylogenies; a task that remains difficult despite recent methodological developments \citep[e.g.][]{pyrondivergence2011,ronquista2012,BEASTmaster}. %Or is it cheap to cite unpublished work (Nick's proto package) in the intro?
% NC: Check Syst Biols policies, but I think it's fine as it's an available package so it'd be like citing something from GitHub. You could ask Nick before publication if there's a different citation he'd prefer you to use.

Up to now, three main approaches have been used to place both living and fossil taxa into phylogenies. These approaches differ mainly in whether they treat fossil taxa as tips or as nodes in the phylogeny, and in which part of the available fossil data is used (i.e. the age of the fossil only or both its age and morphology). Classical cladistic methods use matrices containing morphological data from both living and fossil taxa and treat each taxon as a tip in the phylogeny. Relationships among the taxa are then inferred using optimality criteria such as maximum parsimony \citep{simpson1945}. This approach is commonly used by paleontologists but it ignores the additional molecular data available from living species and does not allow use of probabilistic methods for dealing with phylogenetic uncertainty. Neontologists, on the other hand, more commonly use probabilistic approaches (e.g. Maximum Likelihood or Bayesian methods) based on matrices containing only molecular data from living species. Because fossil taxa do not usually have available DNA, fossils are used as nodes rather than tips in these phylogenies and their occurrence dates are used to time calibrate phylogenies \citep{zuckerkandl1965}. There have been great improvements in the theory and application of these two approaches \citep[e.g.][]{bapsta2013,stadlerdating2013,heaththe2013} as well as much debate about the "best" approach to use \citep[e.g.][]{spencerefficacy2013,wrightbayesian2014}. However neither approach uses all the available data.

A final approach, known as the Total Evidence method, uses matrices containing molecular data from living taxa and morphological data from both living and fossil taxa \citep{eernissetaxonomic1993}. This approach treats every taxon as a tip in the phylogeny, uses the occurrence age of the fossils to time calibrate the phylogeny (known as tip-dating; \citealt{ronquista2012}), and allows the use of probabilistic methods for estimating phylogenetic uncertainty \citep{ronquista2012}. Total Evidence methods have been successfully applied to empirical data \citep[e.g.][]{pyrondivergence2011,ronquista2012,schragocombining2013,slaterphylogenetic2013,beckancient2014}, and are becoming an increasingly popular way of adding fossil taxa to phylogenies. However, although the Total Evidence approach seems very promising, there is one big drawback in using this approach: it requires a lot of data that can be difficult (or impossible) to collect. The morphological data for living taxa are rarely collected when molecular data are available (e.g. \citealt{O'Leary08022013} \textit{vs.} \citealt{meredithimpacts2011}), and for fossil taxa, data can only be collected from features preserved in the fossil record (for example, in vertebrates, the hardest parts of the skeleton are more often preserved than soft parts; \citealt{sansomfossilization2013}). Therefore Total Evidence matrices are likely to contain a lot of missing data that may affect the method's ability to infer correct topologies, branch lengths and support values \citep{salamin2003}. 

%TG: I completely removed this paragraph
%The effect of missing data on phylogenetic inferences has been widely studied \citep{wiensmissing2003,wiensmissing2006,wiensmissing2008,lemmonthe2009,rouresite-specific2011,sansomfossilization2013,pattinsonphylogeny2014,sansombias2014}. Missing molecular data has been seen by some authors as an issue because it can, in some parts of the tree, decrease phylogenetic signal (i.e. the evolutionary information contained within the matrix allowing the inference of topology and branch lengths), especially when using large matrices \citep{lemmonthe2009}. However, this may not be a major issue because phylogenetic signal is easily increased by: (i) including a "modest" number (\texttildelow % NC: This symbol doesn't show up on my PDF. Check.
%50) of highly-covered genes (i.e. a number of genes that are available for all taxa; \citealt{rouresite-specific2011}); (ii) adding a greater number of taxa (especially slowly-evolving taxa or taxa close to the outgroup; \citealt{rouresite-specific2011}); and (iii) choosing more appropriate models of sequence evolution \citep{wiensmissing2006,wiensmissing2008,rouresite-specific2011}. Similarly, missing morphological data might be seen as either a major or minor issue for accurately inferring phylogenies depending on the study in question \citep{wiensmissing2003,sansomfossilization2013,pattinsonphylogeny2014}. Because soft-tissue characters are rarely preserved in the fossil record, missing data is mainly found in these characters, and is therefore not randomly distributed which can lead to biased placement of fossil taxa in phylogenies (e.g. \citealt{sansomfossilization2013}, but see \citealt{pattinsonphylogeny2014}). However, the phylogenetic signal is not related to the amount of missing data \textit{per se} but to the number of informative characters for each taxon, therefore missing data is less of an issue than the number of shared informative characters \citep{kearneyfragmentary2002,wiensmissing2003,pattinsonphylogeny2014}.
% NC: I'm a little dubious of what this paragraph is trying to get at in places. We should discuss at some point. TG: my idea was to do the "literature-review" part of the intro: missing data is already widely studied (answer to the comment: "hun... We already knew that") but it has never been studied in TEM framework. And that's kind of the point we're getting toward with our work: we don't really care about the effect of missing data per se (i.e. what's the threshold where it starts fucking up the phylogeny) but more the fact that Bayesian fails.
% NC: OK fair enough. I think though if this is too long we can probably cut this paragraph and put a bit of it into the paragraph below.

Although missing data does not appear be a major problem in molecular and morphological matrices separately (as long as enough data overlaps in each case) \citep{wiensmissing2003,wiensmissing2006,wiensmissing2008,rouresite-specific2011,pattinsonphylogeny2014}, it may become more of an issue in Total Evidence matrices containing both molecular and morphological data for living and fossil taxa. This may be particularly problematic as fossil taxa (generally) do not have molecular data, resulting in a large section of missing data in Total Evidence matrices. Until now, no attempt has been made to study the impact of this issue on phylogenetic inference using Total Evidence methods.
% NC: THIS SECTION NEEDS WORK!
It is expected that missing in this framework will affect the ability to recover the correct topology as well as the correct branch length estimation \citep{wiensmissing2003,wiensmissing2006,wiensmissing2008,lemmonthe2009,rouresite-specific2011,sansomfossilization2013,pattinsonphylogeny2014,sansombias2014}. In this study, we focus only on the effect of missing data on the ability to recover a correct topology because it is a crucial aspect of a phylogeny in many macroevolutionary studies \citep[e.g.][]{manosphylogeny2007,meredithimpacts2011,jetzthe2012}. %TG or at some more specific example instead of the cite: "In this study, we focus only on the effect of missing data on the ability to recover a correct topology because it is a crucial aspect of a phylogeny in many macroevolutionary studies (e.g. for taxonomical focus \citep{meredithimpacts2011,jetzthe2012}, or for studying evolutionary trends %CITE?\citep[e.g.][]{})"?
Even though branch length is another primordial aspect in phylogenies (namely for timing extinction and/or speciation events \citep[e.g.][]{ronquista2012}), previous studies have empirically assessed the effect of the Total Evidence method on branch length variation \citep{ronquista2012,schragocombining2013,slaterphylogenetic2013,beckancient2014} %TG: add:"by using a fixed topology"?
Moreover, variation in branch length are more complex to simulate and to measure when topology varies.
% NC: I don't get the logic of this part 
% because topology is the first and most straightforward aspect reflecting phylogenetic signal (i.e. topological changes are discrete opposed to branch length changes are continuous). 
% NC: Need a better justification of WHY topology is interesting.

%Although correctly inferring branch lengths is also important, we do not consider the inference of branch lengths here.
% NC: WHY??? Difficulties with simulations? Topology is more important? WHY?
%Also, the effect of Total Evidence method has not been formally assessed in previous studies using fixed topology \citep{ronquista2012,schragocombining2013,slaterphylogenetic2013,beckancient2014}.
% NC: OK this last sentence could be worked in earlier on to explain WHY topology not branch lengths.

Here we use a simulation approach to assess the effect of missing data on tree topologies inferred from Total Evidence matrices. Since the molecular part of a Total Evidence matrix acts like a "classical" molecular matrix containing only the living taxa \citep{ronquista2012}, the effect of missing data on such matrices is well known \citep{wiensmissing2006,wiensmissing2008,rouresite-specific2011}. Therefore, we focus only on missing data in the morphological part of the matrix. We investigate three major parameters that directly affect the completeness of the morphological part of the matrix: (i) the proportion of living taxa with no morphological data; (ii) the proportion of missing data in the fossil taxa; and (iii) the proportion of missing morphological characters for both living and fossil taxa in the matrix. We remove data from a Total Evidence matrix by changing the values of these three parameters and then assess how this affects the topology of trees inferred using Maximum Likelihood and Bayesian methods. We chose these parameters because they reflect empirical biases in data availability. The advent of molecular phylogenetics means that morphological data for living species is rarely collected, and few people have the skills to identify characters needed for detailed phylogenetic analysis. Missing data in fossil taxa is very common due to preservation biases \citep{sansomfossilization2013}, and the overall number of characters depends on the effort of the people identifying them \citep[e.g.][]{O'Leary08022013}. We find that the ability of Total Evidence methods to recover the correct topology is increased when using Bayesian methods. Also minimizing the proportion of living taxa with no morphological data and the proportion of missing morphological characters improves the ability of Total Evidence methods to recover the correct topology more so than minimizing the proportion of missing data in the fossil record.

%---------------------------------------------
%
%       METHODS
%
%---------------------------------------------
 
\newpage

\section{Methods}
To explore how missing data in the morphological sections of Total Evidence matrices influences tree topology, we used the following protocol (note that we explain each step in detail below this general outline; Fig.~\ref{Fig_Outline}).
% NC: I've added numbers in here (125 matrices etc) so it matches the figure legend better. Can you check I've put the right numbers in? I think it may be 124 in some cases...
\begin{enumerate}
\item{Generating the matrix:} \label{step:generate_matrix} \\
We randomly generated a birth-death tree (hereafter called the "true" tree) and used it to infer a matrix containing both molecular and morphological data for living and fossil taxa (hereafter called the "complete" matrix).
\item{Removing data:} \label{step:remove_data} \\
We removed data from the morphological part of the "complete" matrix to simulate the effects of missing data by modifying three parameters (i) the proportion of living taxa with no morphological data ($M_{L}$), (ii) the proportion of missing data in the fossil taxa ($M_{F}$) and (iii) the proportion of missing morphological characters ($M_{C}$). The resulting 125 matrices are called hereafter "missing-data" matrices.
\item{Inferring phylogenies:} \label{step:build_phylo} \\
We inferred phylogenetic trees from the "complete" matrix and from the 125 "missing-data" matrices resulting in one tree generated from a matrix containing no missing data (hereafter called the "best" tree) and 125 trees inferred from the matrices with missing morphological data (hereafter called the "missing-data" trees). Phylogenies were inferred via both Maximum Likelihood and Bayesian approaches.
\item{Comparing topologies:} \label{step:compare_topo} \\
We compared the "best" tree to the "missing-data" trees to assess the influence of each parameter ($M_{L}$, $M_{F}$, $M_{C}$) and their interactions on the topologies of our phylogenies
\end{enumerate}
We repeated these four steps 50 times to account for variation in our random parameters in the simulations.

\begin{figure}
\centering
    \includegraphics[keepaspectratio=true]{Figures/In_main/Simulations_outline-BW.pdf}
\caption{Protocol outline.
(1) We randomly generated a birth-death tree (the "true" tree) and used it to infer a matrix with no missing data (the "complete" matrix).
(2) We removed data from the morphological part of the "complete" matrix resulting in 125 "missing-data" matrices.
(3) We built phylogenetic trees from each matrix using both Maximum Likelihood and Bayesian methods.
(4) We compared the "missing-data" trees to the "best" tree.
We repeated these four steps 50 times.}
\label{Fig_Outline}
\end{figure}
% NC: Can you centre these around the boxes? and make the comparing topologies arrow touch the two triangles? I'll show you what I mean. 
% TG: I have different versions for the figure available (you can see them opening it with inkscape I think - else I'll show you).
% NC: Also check the figure is in a format and of correct quality for Syst Biol.

%---------------------------------------------
%       1-Generating the matrix
%---------------------------------------------

\subsection{1-Generating the matrix} % NC: Can you number these subsections like they are in the steps above? % TG: 
First we randomly generated a "true" tree of 50 taxa in R v. 3.0.2 \citep{R302} using the package diversitree v. 0.9-6 \citep{fitzjohndiversitree2012}. We generated the tree using a birth death process by sampling speciation ($\lambda$) and extinction ($\mu$) rates from a uniform distribution but maintaining $\lambda$ $>$ $\mu$ \citep{paradistime-dependent2011}. We implemented a rejection sampling algorithm to select only trees with 25 living and 25 fossil taxa to ensure that we had enough taxa of each type for our missing data simulations to work. We then added an outgroup to the tree, using the mean branch length of the tree to separate the outgroup from the rest of the taxa, and with the branch length leading to the outgroup set as the sum of the mean branch length and the longest root-to-tip length of the tree.

Next, we generated a molecular and a morphological matrix from the "true" tree. The molecular matrix was inferred from the "true" tree using the R package phyclust v. 0.1-14 \citep{chen2011}. The matrix contained 1000 character sites for 51 taxa and was generated using the seqgen algorithm \citep{ranbaut1997seqgen} and using the HKY model \citep{HKY85} with random base frequencies and transition/transversion rate of two \citep{douadycomparison2003}. The substitution rates were distributed following a gamma distribution with an alpha ($\alpha$) shape of 0.5 \citep{yangamong-site1996}. We chose a low value of $\alpha$ to reduce the number of sites with high substitution rates, thus avoiding too much homoplasy and a decrease in phylogenetic signal. We selected the parameters above to generate data with no special assumption about how the characters evolved, and to reduce the computational time required if these parameters were estimated rather than defined in the tree building part of the analysis (even with the parameters defined, total computational time for the whole analysis was around 150 CPU years). All the molecular information for fossil taxa was replaced by missing data ("?").

We inferred the morphological matrix using the R package ape v. 3.0-11 \citep{paradisape:2004} to generate a matrix of 100 character sites for 51 taxa. We assigned the number of character states (either two or three) for each morphological character by sampling with a probability of 0.85 for two states characters and 0.15 for three state characters. These probabilities were selected using the overall distribution of character states extracted from 100 published empirical morphological matrices (See \hyperref[SupplementaryMaterial]{Supplementary Material Section 1}). We then ran an independent discrete character simulation for each character using the "true" tree with the character's randomly selected number of states (two or three) and assuming an equal rate of change (i.e. evolutionary rate) from one character state to another \citep{Pagel22011994}. This method allows us to have only two parameters for each character: the number of states and the evolutionary rate. For each character, the evolutionary rate was sampled from a gamma distribution with $\alpha$ = 0.5. We used low evolutionary rate parameters to avoid homoplasy in the morphological part of the matrix and create a clear phylogenetic signal \citep{wagner2000,davalosintegrating2014,wrightbayesian2014}.

Finally, we combined the morphological and molecular matrices obtained from the "true" tree. Hereafter we call this the "complete" matrix, i.e. the matrix with no missing data except for the molecular data of the fossil taxa.

%---------------------------------------------
%       2-Removing data
%---------------------------------------------

\subsection{2-Removing data}
\label{Removing_data}
We modified the "complete" matrix to get matrices with missing data by randomly replacing data with "?" in the morphological part of the matrices according to the following parameters:

\begin{enumerate}
\item{$M_{L}$, the proportion of living taxa with no morphological data: 0\%, 10\%, 25\%, 50\% or 75\%.}
This parameter illustrates the number of living taxa that are present in the molecular part of the matrix but not in the morphological part. This reflects the fact that detailed morphological data for living species are scarce.
% NC: Not sure you can really "blame" this on DNA data availability!
\item{$M_{F}$, the proportion of missing data in the fossil taxa: 0\%, 10\%, 25\%, 50\% or 75\%.}
This parameter illustrates the quality of the fossil record. 
\item{$M_{C}$, the proportion of missing morphological characters for both living and fossil taxa: 0\%, 10\%, 25\%, 50\% or 75\%. }
This parameter illustrates the number of available morphological characters for both living and fossil taxa.
\end{enumerate}

In practice, each parameter represents a different way of removing data from the matrix: $M_{L}$ removes rows from the living taxa's data; $M_{F}$ removes cells from the fossil taxa's data; and $M_{C}$ removes columns across both living and fossil taxa's data. Note that $M_{L}$ and $M_{F}$ differ not only because of the region of the matrix affected: for $M_{L}$ all the morphological data of a percentage of living taxa is removed, whereas for $M_{F}$ a percentage of the data is removed at random from across the whole of the morphological matrix for fossil taxa.

We created matrices using all parameter combinations resulting in 125 ($5^3$) matrices. Note that one of these combinations has no missing data so is equivalent to the "complete" matrix, thus we have one effectively complete matrix in our 125 "missing-data" matrices. Because some parameter combinations introduce a lot of missing data (e.g. $M_L$=75\%, $M_F$=75\% and $M_C$=75\%), some matrices contained fossil taxa without any data at all. When this occurred we repeated the random deletion of characters until each taxon had at least 5\% data across the whole morphological part of the matrix.

%---------------------------------------------
%       3-Building phylogenies
%---------------------------------------------

\subsection{3-Building phylogenies}
From the resulting matrices we generated two types of trees: the "best" tree inferred from the "complete" matrix and the "missing-data" trees inferred from the 125 matrices with various amounts of missing data. The "true" tree was used to generate the "complete" matrix and reflects the "true" evolutionary history in our simulations. The "best" tree, on the other hand, is the best tree we can build using state-of-the-art phylogenetic methods. In real world situations, the "true" tree is never available to us because we cannot know the true evolutionary history of a clade (except in very rare circumstances, e.g. \citealt{rozen2005}). Therefore, here we focus on comparing the trees inferred from the matrices with missing data to the "best" tree, rather than the "true" tree, as the "best" tree is generally what we have to work with.

\subsubsection{Maximum Likelihood}
The "best" tree and the "missing-data" trees were inferred using RAxML v. 8.0.20 \citep{Stamatakis21012014}. For the molecular data, we used the GTR + $\Gamma_4$ model (\citealt{tavare1986}; default GTRGAMMA in RAxML v. 8.0.20; \citealt{Stamatakis21012014}).% as a generalization of the HKY + $\Gamma_4$ model \citep{HKY85}.
%The GTR model can be seen as a generalization of the HKY model (the two parameters from the HKY model are implicitly included in the six from GTR model - \citealt{stamatakisa2008}). - TG: Is that useful? 
% NC: I think most people know that already.
For the morphological data, we used the implemented Markov \textit{k} state model \citep{lewisa2001} %which is a generalization of the JC69 model \citep{jc69} with \textit{k} $\geq$ 2 - TG: same?
% NC: Again leave it out
 assuming an equal state frequency and a unique overall substitution rate ($\mu$) following a gamma distribution of the rate variation with four distinct categories (M\textit{k} + $\Gamma_4$; -K MK option in RAxML v. 8.0.20; \citealt{Stamatakis21012014}).

To measure the phylogenetic signal of our simulations, we first ran a fast bootstrap analysis with 500 replicates on the "complete" matrix. We removed all the simulations with a median bootstrap support lower than 50 as a proxy for weak phylogenetic signal \citep{zanderminimal2004}. We repeated this selection until we obtained 50 sets of simulations (i.e. 50 "complete" and 50 x 125 "missing-data" matrices) with a relative good phylogenetic signal (median bootstrap $>$ 50). On these selected simulations, we used the fast bootstrap algorithm and performed 1000 bootstraps for each tree inference to assess topological support \citep{pattengale2010many}.
%The bootstrap algorithm used in RAxML is the Lazy Sub-tree Rearrangement (LSR) which consists of pruning one sub-tree from the tree and subsequently reinserting it to all neighbouring branches \citep{stamatakisa2008}. Sub-tree Pruning and Reinserting methods (SPR) have been demonstrated to be better than other methods (e.g. Nearest Neighbouring Interchange - NNI) for recovering good bootstrap values \citep{salamin2003}. %TG: The explanations of the algorithm part for the BS was proposed by Trevor I think it's useless
% NC: perhaps somewhere in brackets just mention it's LSR bootstrapping. No need for the rest I agree.
Using these parameters took \texttildelow 8 CPU years to build 50 sets of 125 bootstrapped Maximum Likelihood trees (2.30GHz clock speed nodes).

\subsubsection{Bayesian}
The "best" tree and the "missing-data" trees were inferred using MrBayes v. 3.2.1 \citep{Ronquist2012mrbayes}. We partitioned the data to treat the molecular part as a non-codon DNA partition and the morphological part as a multi-state morphological partition. The molecular evolutionary history was inferred using the HKY model with a transition/transversion ratio of two \citep{douadycomparison2003} and a gamma distribution for the rate variation with four distinct categories (HKY + $\Gamma_4$). For the morphological data, we used the Markov \textit{k} state model \citep{lewisa2001}, with equal state frequency and a unique overall substitution rate ($\mu$) with four distinct rates categories (M\textit{k} + $\Gamma_4$). We chose these models to be consistent with the parameters used to generate the "complete" matrix.

Each Bayesian tree was estimated using two runs of four chains each for a maximum of 5$\times$$10^7$ generations. We used the following priors for each tree: (i) the "true" treeâ€™s topology as a starting tree (with a starting value for each branch length of one), (ii) an exponential prior on the shape of the gamma distribution of $\alpha$ = 0.5 for both partitions, and (iii) a transition/transversion ratio prior of two sampled from a strong beta distribution ($\beta$(80,40)). We used these priors to speed up the Bayesian estimation process. These priors biased the way the Bayesian process calculated branch lengths by giving non-random starting points and boundaries for parameter estimation however, here we are focusing on the effect of missing data on tree topology and not branch lengths. Even using these priors, it took \texttildelow 140 CPU years to build 50 sets of 125 Bayesian trees (2.30GHz clock speed nodes). The detailed MrBayes parameters are available in the \hyperref[SupplementaryMaterial]{Supplementary Material S1}.

We used the average standard deviation of split frequencies (ASDS) as a proxy to estimate the convergence of the chains and used a stop rule when the ASDS went below 0.01 \citep{Ronquist2012mrbayes}. The effective sample size (ESS) was also checked on a random sub-sample of runs in each simulation to ensure that ESS $>>$ 200 \citep{drummond2006ess}. Finally we built the Bayesian consensus tree from the combined chains excluding the 25\% first iterations (burn-in) with a strict majority rule \citep{Ronquist2012mrbayes}.
%---------------------------------------------
%       4-Comparing Topologies
%---------------------------------------------

\subsection{4-Comparing topologies}
We compared the topology of the "missing-data" trees to the "best" tree to measure the effect of the three parameters $M_{L}$, $M_{F}$ and $M_{C}$ on tree topology. We used the Robinson-Foulds distance \citep{RF1981} to assess the amount of conserved clade positions and the Triplets distance \citep{dobson1975triplets} to assess the amount of wildcard taxa  (i.e. taxa that frequently change position in different trees - \citealt{kearneyfragmentary2002}). We normalised both metrics using the Normalized Tree Similarity index \citep{Bogdanowicz2012} to generalize our results for any \textit{n} number of taxa. These metrics are described in detail below.

\subsubsection{Robinson-Foulds distance}
The Robinson-Foulds distance \citep{RF1981}, or "path difference", measures the number of shared clades across two trees. The metric reflects the distance between the distributions of tips among clades in the two trees (\citealt{RF1981}; see \hyperref[SupplementaryMaterial]{Supplementary Material S2} for calculation details). This metric is bounded between one, when the two trees are identical, and $n-2$ (for two trees with $n$ taxa) when there is no shared clade in the two trees. This metric is sensitive to minor changes in clade conservation: if the trees are composed of two clades of three taxa (\textit{(((a,b),c),((d,e),f))}), the swapping of any two taxa will lead to a maximal score of the Robinson-Foulds distance indicating poor tree similarity. We normalised this metric using the Normalized Tree Similarity index \citep{Bogdanowicz2012}. This index allows us to compare the metric for any number of taxa by scaling it with the expected value of the metric when comparing two random trees (see \hyperref[SupplementaryMaterial]{Supplementary Material S2} for the calculation details). Therefore, when the Normalised Robinson-Foulds distance between two trees is equal to one, the trees are identical; if the Normalised Robison-Foulds distance is equal to zero, the trees are no more different than expected by chance; finally if the Noramlised Robison-Foulds distance is lower than zero, the trees are more different than expected by chance.

\subsubsection{Triplets distance}
The Triplets distance \citep{dobson1975triplets} measures the number of sub-trees made up of three taxa that differ between two trees (\citealt{critchlowthe1996}; see \hyperref[SupplementaryMaterial]{Supplementary Material S2} for calculation details). This metric measures the position of each taxon and clade in relation to its closest neighbours. It is bounded between 0 when the two trees are identical and $\binom{n}{4}$ (for two trees with $n$ taxa) when there is no shared taxon/clade position in the two trees. Therefore this metric sensitive to the conservation of wildcard taxa. We normalised this metric in the same way as we did for the Robison-Foulds distance resulting in the Normalised Triplets distance.

%\subsubsection{Normalized Tree Similarity}
%The Normalized Tree Similarity index, $NTS_m$ (\citealp{Bogdanowicz2012}) allows us to compare the two metrics above for any number of taxa, by scaling the value of the metric $m$ (either Robinson-Foulds or Triplets distance in our study) to the expected value of the metric $m$ when comparing two random trees (see \hyperref[SupplementaryMaterial]{Supplementary Material S2}). When $NTS_m$=1, the two trees are identical, when $NTS_m$=0 the trees are no more different than expected when comparing two random trees, and when $NTS_m$$<$0, the difference between the two trees is greater than when comparing two random trees. In our study we used the $NTS_m$ index as a proxy for topological similarity: a high score of this index (i.e. towards one) means that the topology is highly conserved between the two trees; conversely, a low score of this index (i.e. towards 0) means that the topological difference between the two trees is as great as expected when comparing two random trees. Note that the two tree comparison metrics represent different aspects of topological similarity and do not perform equally well (see \hyperref[metrics_discussion]{discussion below} and \citealt{kuhnerpractical2014}).

\subsubsection{Paired trees comparisons} % NC: OK I'm lost in this section! We should discuss then think about how to express this in an understandable fashion. TG: or no subheading at all?
\label{tree_comparisons}
For the Maximum Likelihood and Bayesian consensus
% NC: is this actually a maximum clade credibility tree - TG: better stick with consensus as described above?
trees we performed pairwise comparisons between the "best" tree and each "missing-data" tree using both the Normalised Robinson-Foulds and Normalised Triplets metrics with the TreeCmp java script \citep{Bogdanowicz2012} resulting each time in 125 Normalised Robison-Foulds distances and 125 Normalised Triplets distance. %For each metric, we then normalized the value using the Normalized Tree Similarity scaled by the mean value of 1000 pairwise random tree comparisons for the metric in question and n = 51 taxa (see \hyperref[SupplementaryMaterial]{Supplementary Material Section 2}). % TG: That is now described in the RF distance subsection just above
%We compared each "missing-data" tree with the "best" tree for each of our 50 simulation runs resulting in 50 comparisons for each "missing-data" tree. We calculated the mode and the 50\% and 95\% confidence intervals from the resulting distribution using the hdrcde R package v3.1 \citep{hdrcde}. TG: explained later on
Also, to take into account the uncertainty of tree inference in both method, instead of using the single Maximum Likelihood and Bayesian consensus trees as above, we extracted 1000 random bootstrapped trees from the Maximum Likelihood analysis and 1000 from the posterior tree distribution of the Bayesian analysis for the "best" trees and then did the same for the 125 "missing data" trees (resulting in 1000 "best" trees and 125$\times$1000 "missing data" trees). 
For a given set of 1000 "missing data" trees and the 1000 "best" trees, we sampled one "missing data" tree and one "best" tree at random and compared them using both the Normalised Robinson-Foulds and Normalised Triplets metrics as described above. We repeated this 1000 times for each set of "missing data" trees resulting in 125$\times$1000 values for each metric. We repeated all the paired trees comparisons described above for each of the 50 simulations. We then calculated the mode and the 50\% and 95\% confidence intervals from the resulting distribution using the hdrcde R package v. 3.1 \citep{hdrcde}. %Finally we tested for significant differences among the different distributions of trees using non-parametric group and pairwise comparisons by using respectively Kruskal-Wallis and Nemenyi-Damico-Wolfe-Dunn test (CITE R PACKAGE)\citet{ruxtontime2008}. TG: finally we didn't (Andrew's advice - see Battacharyya)

\subsubsection{Summarizing the comparisons}
Finally after doing all theses %stupid
50 simulations and comparisons within, we tested the effects of our missing data parameters ($M_{L}$, $M_{F}$ and $M_{C}$) on the ability to recover the correct topology in a Total Evidence framework. Additionally, we also assessed the effect of the different methods (i.e. Maximum Likelihood, Bayesian consensus, Bootstraps and Bayesian posterior trees distribution).% and the joint effect of the our missing data parameters and the methods.
% TG: the following part might be useless (complete repetition of above)
To assess the ability to recover the "correct" tree topology on our 50 simulations, we used the scores for both normalised metrics (Robinson-Foulds and Triplets distance). We looked at the following effects:
\begin{enumerate}
\item{The three missing data parameters ($M_{L}$, $M_{F}$ and $M_{C}$);}
\item{The methods (Maximum Likelihood, Bayesian consensus, Bootstraps and Bayesian posterior trees distribution);}
%\item{The joint effect of the missing data parameters and the methods.} actually this is implied in both already.
\end{enumerate}
% TG: end of the repetition

%Introducing Bhattacharyya Coefficient
To evaluate these effects we measured the similarities among the different normalised metrics scores (Robinson-Foulds and Triplets distance) for all the analysis above using the Bhattacharyya Coefficient \citep{Bhattacharyya}. The Bhattacharyya Coefficient is the probability of overlap between two distributions, ranging from 0 to 1 (\citealt{Bhattacharyya} - see \hyperref[SupplementaryMaterial]{Supplementary Material Section 2} for calculation details).

%Explaining the parameter pairwise comparisons (ANOVA equivalent=within)
For assessing the effect of our three missing data parameters ($M_{L}$, $M_{F}$ and $M_{C}$), we calculated the Bhattacharyya Coefficient between each pairwise combination for each parameter state (0\%, 10\%, 25\%, 50\%, 75\%). This resulted in 7875 pairwise distributions pairwise comparisons (triangular $3^5$$\times$$3^5$ matrix). We represented the results in a triangular matrix with the values of each pairwise Bhattacharyya Coefficient coloured according to their values (in green when the distribution overlap completely (Bhattacharyya Coefficient=1) and in red when they don't (Bhattacharyya Coefficient=0) - Fig.~\ref{Fig_Results-paircomp_within}). We repeated this procedure for all the methods (Maximum Likelihood, Bayesian consensus, Bootstraps and Bayesian posterior trees distribution).

%Explaining the methods comparisons (ANOVA equivalent=among)
Finally, for assessing the effect of the method on the ability to recover the "correct" topology, we calculated the Bhattacharyya Coefficient between each 125 normalised metrics scores (Robinson-Foulds and Triplets distance) for the 6 pair of methods (e.g. Maximum Likelihood\textit{versus} Bayesian consensus). We then reported the minimum, the 1st quartile, the median, the mean, the 3rd quartile and the maximum of the probabilities of overlap between the methods (i.e. the Bhattacharyya Coefficients) in table ~\ref{Tab_Results-Difference_methods}. When two methods display similar normalised metrics scores for each comparison between the "best" tree and the "missing data" trees, we expect the Bhattacharyya Coefficients to be close to 1. On the opposite, if the two methods displays dissimilar normalised metrics scores, we expect the Bhattacharyya Coefficients to be close to 0. 

%---------------------------------------------
%
%       RESULTS
%
%---------------------------------------------

\section{Results}

% NC: Split into three sections to answer three questions

%\subsection{The effects of missing data on tree topology}

\begin{figure} 
\centering
    \includegraphics[width=1\textwidth]{Figures/In_main/AllMethods-RF+Tr-colour.pdf}
\caption{Comparison between the effect of missing data and the tree inference method on topological recovery. The amount of missing data for each parameter is represented on the x axis. The difference in topologies is represented on the y axis, using standardized Robinson-Foulds distance (upper row) and standardized Triplets distance (lower row). Points represent the modal value of each distribution; thick solid and thin dashed lines represents the 50\% and 95\% confidence intervals or the distributions respectively. Analyses using the Maximum Likelihood trees are represented in black, the Bayesian consensus %MCC?
trees in red, the Maximum Likelihood bootstrap trees in green and the Bayesian posterior tree distribution in blue.}
\label{Fig_Results-permeth_perparam} %Differences within a subset of parameters and between methods
\end{figure}

% NC: Y axes should be standardised Robinson Foulds
% NC: X-axes should have (Mc) etc in brackets after the names
% NC: Also should probably have the X axes read : Missing character (Mc, %) and remove % from tick labels

%NC: I would rewrite these figure labels as follows. I prefer the details of colours and things to be in brackets not in the text. Rewrite the others in a similar fashion.
%NC: Remeber it's not topology, but topological recovery that you are measuring
%The effects of increasing missing data on topological recovery using Maximum Likelihood trees (black), Bayesian consensus trees (red), Maximum Likelihood bootstrap trees (green) and Bayesian posterior tree distributions (blue). The percentage of missing data for each parameter (Mc, ML, MF) is shown on the x axis. Topological recovery was measured using two different tree comparison metrics: standardized Robinson-Foulds distance (upper row) and standardized Triplets distance (lower row). The graph shows the modal value (points), the 50\% and 95\% confidence intervals (thick solid lines) and the distributions (thin dashed lines), of the tree comparison metric for each missing data parameter and tree inference method. 

As the amount of missing data in the morphological part of the total evidence matrix increases, our ability to recover the correct topology decreases, regardless of the missing data parameter ($M_{L}$, $M_{C}$ or $M_{F}$), the method of tree inference (Maximum Likelihood or Bayesian) or the tree comparison metric used (standardized Robinson Foulds or standardized Triplets distance). 
% NC: Results are for YOUR results. compare it to others in the Discussion only
% NC: I've rephrased below as it was getting confused between what is GOOD and BAD for topological recovery and the difference between the parameters and the methods. I've tried to clarify.
% NC: Note that you need to be clear what the methods are outperforming...
However, the different missing data parameters and tree inference methods do not affect the topology in the same way (Fig.~\ref{Fig_Results-permeth_perparam}). % And figure 3 too right?

%Increasing the amount of missing data in the living taxa ($M_{L}$) has the worst effect on topological recovery, though only when using the standardized Robinson Foulds distance. Using the standardized Robinson Foulds distance, which represents clade conservation between trees, we get the best topological recovery using Bayesian consensus trees. However, using standardized Triplets distance, which represents the movement of wildcard taxa, Maximum Likelihood outperforms the other methods but with wider distributional overlap (Fig.~\ref{Fig_Results-permeth_perparam}). %NC: What distributions? Do you need to say that?

\begin{figure} 
\centering
    \includegraphics[width=1\textwidth]{Figures/In_main/ML+Baycon-AllParam-RF+Tr-BW.pdf}
\caption{Trend of the effect of missing data on topological recovery on the Maximum Likelihood and the Bayesian consensus trees. The amount of missing data per parameter ($M_{L}$, $M_{F}$ and $M_{C}$) is represented along the x axis. The colour gradient from white to black represents respectively, 0\%, 10\%, 25\%, 50\% and 75\% of missing data for each parameter. The topology is represented on the y axis, both using Robinson-Foulds distance (upper row) and Triplets distance (lower row). Points represent the modal value of each distribution ; thick solid and thin dashed lines represents respectively the 50\% and 95\% confidence intervals or the distributions. The Maximum Likelihood trees are represented in black and the Bayesian consensus trees in grey.}
\label{Fig_Results-global_perparam} %Differences between all the parameters and between two methods (ML vs BC)
\end{figure}

%When looking at the effect of all parameters combinations on the Bayesian consensus trees, this method outperforms Maximum Likelihood for the Robinson-Foulds distance and plateaus to a minimal modal tree similarity of 0.69 regardless the amount of missing data (Fig.~\ref{Fig_Results-global_perparam} - see also \hyperref[SupplementaryMaterial]{Supplementary Material Section 3}). However, regarding the Triplets distance, Bayesian consensus trees seems to perform poorly but with great confidence intervals overlaps (Fig.~\ref{Fig_Results-permeth_perparam} and Fig.~\ref{Fig_Results-global_perparam}). These larger confidence intervals are due to the lower ability of the Triplet method to highlight precise differences in topology \citep{kuhnerpractical2014}. Results for the global trend comparison between the uncertainty methods (Bootstrap and Bayesian posterior tree distribution) are available in the supplementary materials (see \hyperref[SupplementaryMaterial]{Supplementary Material Section 3}).
% NC: Move what you need of this to the sections below.
% NC: Remember you're not testing the RF and TD, you're seeing that the loss of data affects clade conservation and wildcard taxa separately.

\subsection{Effects of the missing data parameters on topological recovery}

%Hide this figure for 'fast' build
%\begin{figure} 
%\centering
%    \includegraphics[width=1\textwidth]{Figures/In_main/PairwiseComp-Baycon-RF+Tr-colour.pdf}
%\caption{Pairwise Bhattacharyya Coefficients within the Bayesian consensus trees. The pairwise trees comparisons are represent on both axis. The colour gradient from white to black represents respectively, 0\%, 10\%, 25\%, 50\% and 75\% of missing data for each parameter. The matrix represents the values of pairwise Bhattacharyya Coefficients going from green (1) to red (0). A. Results for the Normalised Robinson-Foulds distance. B. Results with the for Normalised Triplets distance.}
\label{Fig_Results-paircomp_within}
%\end{figure} %Pairwise BC for the Bayesian consensus trees for RF and TR. (parameters differences)

%Increasing the amount of missing data in the living taxa ($M_{L}$) has the worst effect on topological recovery, though only when using the standardized Robinson Foulds distance.

%NC: I'd start by describing the pattern with reference to figs 2 and 3 and then the "significant" results

When looking at the pairwise Bhattacharyya Coefficients between all the parameters combinations for Bayesian consensus trees, % NC: Instead just state the result. "We found that number of missing living taxa ($M_{L}$) results the lowest probabilities of overlap (table X)". In the same way as you wouldn't say "Using a linear regression we found that x ~ Y." You'd say "X and Y were significantly correlated (Table1)".
%NC: Also what IS the result really? You find that ML overlaps less with the other parameters so it's significantly different in it's effect? And from the graph you can see it's worse? What does oerlap MEAN in terms of the answer to the question, i.e. how do the parameters affect topological recovery. 

the number of missing living taxa ($M_{L}$) results the lowest probabilities of overlap (right lower corner in Fig.~\ref{Fig_Results-paircomp_within}-A). However, when looking at the Triplets distance, because of the high overlap in confidence intervals, there is no major region with low probability of overlap. One should note however that the bottom line of the matrix where the Bhattacharyya Coefficients are very low is due to comparing the distribution of the pairwise comparison of the "best" tree against the "missing-data" tree with no missing data ($M_{L}$=0\%, $M_{C}$=0\%, $M_{F}$=0\%) , leading to a normalised Triplets score of 1 every time (with no variance, see Fig.~\ref{Fig_Results-permeth_perparam} and Fig.~\ref{Fig_Results-global_perparam}). The pairwise comparisons for the other methods (Maximum Likelihood, Bootstraps and Bayesian posterior trees) are available in the supplementary materials (see \hyperref[SupplementaryMaterial]{Supplementary Material Section 3}).

\subsection{Effects of the tree inference method on topological recovery}

%NC: I'd start by describing the pattern with reference to figs 2 and 3 and then the "significant" results

%\begin{figure} 
%\centering
  %  \includegraphics[width=1\textwidth]{Figures/In_main/BC-MethodComp-RF+Tr-colour.pdf}
%\caption{Distribution of the Bhattacharyya Coefficients between methods. Curves represents the kernel density estimations of the Bhattacharyya Coefficients between the four different methods. A. Results for the Normalised Robinson-Foulds distance. B. Results with the for Normalised Triplets distance. Pikes around the two different extremes values of the Bhattacharyya Coefficients show the two pairs of most dissimilar methods (Bayesian consensus vs. Bootstraps and Bayesian consensus vs. Bayesian posterior trees) for the Normalised Robinson-Foulds distance ; and the most similar methods (Bootstraps and Bayesian posterior trees) for the Normalised Triples methods.}
%\label{Fig_Results-paircomp_without} %Comparisons between methods, both in RF and Triplets.(method differences)
%\end{figure}

\begin{table}[ht]
\label{Tab_Results-Difference_methods}
\caption{Bhattacharyya Coefficients of the pairwise method comparisons, each of which corresponds to the normalised distance between the "best" tree and the "missing data".}
\centering
\begin{tabular}{lccccccc}
  \hline
 Comparison &  Metric & Min. & 1st Qu. & Median & Mean & 3rd Qu. & Max. \\ 
  \hline
    Maximum Likelihood \textit{vs.} & $RF^1$ & 0.00 & 0.00 & 0.10 & 0.20 & 0.32 & 1.00 \\ 
    Bayesian consensus              & $Tr^2$ & 0.34 & 0.49 & 0.61 & 0.62 & 0.75 & 1.00 \\ 
    Maximum Likelihood \textit{vs.} & $RF^1$ & 0.03 & 0.54 & 0.69 & 0.64 & 0.77 & 0.98 \\ 
    Bootstraps                      & $Tr^2$ & 0.08 & 0.57 & 0.65 & 0.64 & 0.73 & 0.82 \\ 
    Maximum Likelihood \textit{vs.} & $RF^1$ & 0.02 & 0.74 & 0.80 & 0.79 & 0.89 & 0.98 \\ 
    Bayesian posterior trees        & $Tr^2$ & 0.21 & 0.67 & 0.73 & 0.72 & 0.77 & 0.84 \\ 
    Bayesian consensus \textit{vs.} & $RF^1$ & 0.00 & 0.00 & \textbf{0.00} & \textbf{0.01} & 0.01 & 0.04 \\ 
    bootstraps                      & $Tr^2$ & 0.08 & 0.38 & 0.59 & 0.57 & 0.73 & 0.84 \\ 
    Bayesian consensus \textit{vs.} & $RF^1$ & 0.00 & 0.00 & \textbf{0.01} & \textbf{0.02} & 0.04 & 0.11 \\ 
    Bayesian posterior trees        & $Tr^2$ & 0.21 & 0.36 & 0.56 & 0.55 & 0.74 & 0.87 \\ 
    Boostraps \textit{vs.}          & $RF^1$ & 0.50 & 0.77 & \textbf{0.85} & \textbf{0.85} & 0.96 & 1.00 \\ 
    Bayesian posterior tree         & $Tr^2$ & 0.91 & 0.96 & \textbf{0.98} & \textbf{0.97} & 0.99 & 1.00 \\ 
   \hline
\end{tabular}
   $^1RF$=Normalised Robison-Fould distance ; $^2Tr$=Normalised Triplets distance
\end{table}

%When looking at the comparisons between the methods 
using the Robinson-Foulds distance, the Bayesian consensus trees have the least overlap with respectively the Bayesian posterior trees and the Bootstrap (Fig.~\ref{Fig_Results-paircomp_without}-A). Using the Triplets distance, the two methods that have the most distribution %NC: ???
are the Bootstraps and the Bayesian posterior trees (Fig.~\ref{Fig_Results-paircomp_without}-B). These results are due to the difference in comparing topologies between Maximum Likelihood and Bayesian consensus trees and Bootstraps and Bayesian posterior trees: the latter two are based on a random pairwise comparison of a sub-sample of 1000 trees. This is likely to add noise to the data.

\subsection{Interactions among missing data parameters and the tree inference method on topological recovery}

%NC: Maybe don't need this section. See how it goes.

% NC: Make sure you're consistent in how you refer to things ie Maximum Likelihood bootstrap trees, not Bootstraps
%---------------------------------------------
%
%       DISCUSSION
%
%---------------------------------------------

% NC: Quick comment. You find that Bayesian consensus trees are "best". BUT I think you need to be careful here with what you mean by "best". They are most likely to recover a more similar topology to the best tree than if you do the crazy tree comparison pairs. But that doesn't mean they're best, just that they perform best in your simulations. They are only best for that. If you want to advise people to use a method that best recovers topology as measured using RF or Triplets, then Bayesian consensus is best. BUT generally that is not what people want. Probably need to think some more about this.

\section{Discussion}

% NC: Discussions usually begin with a paragraph that sums up the results. Then you can move on to discussing caveats, problems, extensions etc. You also need to show how your work fits in with the wider literature. So I think you need to re-write this completely with that in mind. Do the same as we did for the introduction - write a line for each paragraph. This reads like you jumped in without thinking clearly about the structure. It probably doesn't need to have all the same subheadings either.
% TG: Ok, from here, everything is new, I tried way more to write something proper following your "results-caveats-problems-extension" structure and I heavily based the structure of the paragraphs on the longevity paper + others (you might recognise you're style in the first paragraph if I did well). I left in comments the skeleton of the structure that I used to help me writing it.
% TG: Also here's a note on my syntax notation for linking the paragraphs: "$"=end OR "^"=start of paragraph number 1. "a >..."=which finsih with the statement OR ">... b"=which follows with the statement b
% TG: Here's the tree-style "plan" I was using (linked with the intro):
% INTRO:
% Â§1 We need fossils
%   |
%   \-> Â§2 How can we use them?
%       |
%       \-> Â§3 Total Evidence method of course!
%           |
%           \-> Â§4 But this method has a lot of missing data?
%               |
%               \-> Â§5 So what do we know about missing data from literature?
%                   |
%                   \-> Â§6 How will that effect TEM?
%                      |
%                      \-> Â§7 We use simulations to test!
%                       
% METHODS
% RESULTS
% DISCUSSION
%       
% Â§1 What do we found (Intro-Â§6)
%   |
%   \-> Â§2 How does that link to what we know (Intro-Â§5)
%       |
%       \-> Â§3 The effect of the parameters
%       |
%       \-> Â§4 The effect of the methods
%       |
%       \-> Caveats
%           |
%           \-> Â§5 The metrics
%           |
%           \-> Â§6 The simulations
%           |
%           \-> Â§7 How to fix them
%           |
%           \-> Â§8 How we can use fossils (Intro-Â§2)

% CONCLUSION.
% part 1- Fossils are important and can be used combined with living taxa (Intro-Â§1 & Â§2)
% part 2- This is how we can improve the data to use them in the best way possible (Conclusion-Â§1 & Â§8)

%Â§1
%Results: global
%   -Missing data increase leads to decreases in recovering the correct topology
%The results of our simulations are in adequacy with recent analysis on the effect of missing data in morphological matrices \citep{pattinsonphylogeny2014,wrightbayesian2014,sansombias2014}. 
Our results show that our ability to recover the correct topology decreases with increasing missing data, regardless of the missing data parameter ($M_{L}$, $M_{C}$ or $M_{F}$), the method of tree inference (Maximum Likelihood or Bayesian) or the tree comparison metric used (standardized Robinson Foulds or standardized Triplets distance). 
%Results 1
%   -However, parameters have different effect
However, within a tree inference method and a metric, the parameters used to remove data have different effects on the ability to recover the correct topology.
%Results 2
%   -Additionally, methods and metrics are unequal
Additionally, the effect of missing data on topological recovery varies with different methods to infer topology and different metrics to compare topologies.
%Wrap up results
%   -Thus, Baycon is better with loads of living 
Thus, we found that using the topologies of Bayesian consensus trees performed consistently better than other methods for recovering the correct topology (Fig.~\ref{Fig_Results-global_perparam} and Fig.~\ref{Fig_Results-paircomp_within}) and that both the amount of available characters ($M_{C}$) and the amount of data for the living taxa ($M_{L}$) are crucial missing data parameters (Fig.~\ref{Fig_Results-paircomp_within}).

% $Â§1: What we found >...
% ^Â§2: >... links with previous studies

%Â§2
%Results 1 from litterature
%   -Missing data per se is not a problem, problem is in the amount of overlap
As proposed in previous studies, the amount of missing data is not a problem \textit{per se} as long as enough data overlaps in the matrix \citep[e.g.][]{kearneyfragmentary2002,wiensmissing2003,rouresite-specific2011,pattinsonphylogeny2014}. %In fact, as shown previously, the amount of overlap in the matrix is more crucial than the proportion of missing data \citep[e.g.][]{rouresite-specific2011}.
%Proposed hypothesis for result 1
%   -One can increase overlap by adding living taxa
To improve the ability to recover the "correct" topology in a Total Evidence framework, one can efficiently increase the amount of overlapping data by reducing the amount of missing data in each of our three selected parameters. In our simulations, the overlap in data is greatly reduced when different parameters are combined with a high amount of missing data (e.g. $M_{L}$=75\%, $M_{F}$=75\% and $M_{C}$=75\% - Fig.~\ref{Fig_Results-permeth_perparam} and Fig.~\ref{Fig_Results-global_perparam}). In this scenario the overall number of missing characters ($M_{C}$) affects the proportion of overlapping data by reducing the overall size of the matrix (Fig.~\ref{Fig_Results-global_perparam}). However it is important to note that in the pairwise Bhattacharyya Coefficient analysis, the area with the lowest probability of distributions overlap is led by comparing the trees with no missing data for the living taxa to the trees with 75\% missing data for the living data (right lower corner in Fig.~\ref{Fig_Results-paircomp_within}).

% $Â§2: There is a different effect form each parameter >...
% ^Â§3: >... because each parameter tells a different story.

%Â§3
%Discussing results 1
This variable effect of the missing data parameters on the ability to recover the "correct" topology (Fig.~\ref{Fig_Results-paircomp_within}) can be explained by the way our parameters are illustrating the sources of missing data in empirical matrices: (1) non coded-living  taxa ($M_{L}$), (2) the quality of the fossil record ($M_{F}$) and (3) the amount of coded characters ($M_{C}$ - see \hyperref[Removing_data]{Methods}). We have suggestions for practical ways to reduce this missing data problem in empirical matrices:
%   -Which parameters does what?
\begin{enumerate}
\item{$M_{L}$ (non coded living taxa):}
The vast majority of known living species have specimens preserved in natural history museums. This is a significant resource and more research effort is needed to code morphological characters in these specimens.
\item{$M_{F}$ (quality of the fossil record):}
The amount of missing data in the fossil record solely depends on the preservation quality of the fossils and can not be actively improved. However, gaps in the matrix can be filled with the discoveries exceptional preserved fossil \citep[e.g.][]{nithe2013}. Fortunately, this parameter is the most difficult to improve but it also has the least influence on whether the simulations recover the "correct" topology (Fig.~\ref{Fig_Results-paircomp_within}).
\item{$M_{C}$ (amount of coded characters):}
The amount of time it takes to build a morphological matrix increases directly with the number of characters involved. Improvement in both decreasing the time taken to build the matrix and increasing the number of characters can be done through vast collaborative data collection projects through web portals such as \textit{morphobank} \citep{morphobank}.
\end{enumerate}
Fixing these three major issues can lead to a sensible increase in the ability to recover the "correct" topology.

% $Â§3: This is how we can fix the parameters >...
% ^Â§4: >... but the methods also influence a lot the ability to recover the right topology.

%Â§4
%Result 2 from literature.
%   -Probabilistic method always better (wright but see spencer) - Check McGuire[nope]/Sean[nope] literature for Bayesian vs. ML?
Nonetheless, the variation in the ability to recover the "correct" topology does not only depend on the missing data in morphological part of the matrix but also heavily on the method used to infer the topology (Fig.~\ref{Fig_Results-permeth_perparam} and Fig.~\ref{Fig_Results-global_perparam}). Previous studies have shown some superiority on methods using simple probabilistic models such as the M\textit{k} model \citep{lewisa2001} over cladistic methods (\citealt{wrightbayesian2014} but see \citealt{spencerefficacy2013}). However, this is the first study, to our knowledge, that compares the performance of the M\textit{k} model \citep{lewisa2001} between Maximum Likelihood and Bayesian inference in a Total Evidence framework. %TG: I guess it's a good selling point: we ARE the first ones to do a sensitivity analysis in a TEM framework (as far as I know).
%Discussing results 2
%   -Why is Baycon better than the others?
When using the clade conservative Robinson-Foulds distance (see \citealt{kuhnerpractical2014} and \hyperref[metrics_discussion]{discussion below}), the Bayesian consensus trees greatly outperforms the Maximum Likelihood trees in the ability to recover the correct tree topology (Fig.~\ref{Fig_Results-global_perparam}). In fact, regardless of the amount of missing data, this method always displays the highest normalised Robinson-Foulds score (always $>$ 0.69 - see \hyperref[SupplementaryMaterial]{Supplementary Material Section 3}). Also, when looking at the pairwise Bhattacharyya Coefficients within each method, the Bayesian consensus tree one display the highest probability of overlap between the results in any case (see \hyperref[SupplementaryMaterial]{Supplementary Material Section 3}).
%   -Why is Baycon so different than Baytre? Since Baycon is calculated from Baytre.
When comparing this to the probability of overlap between the methods, it is counter intuitive to note that the Bayesian consensus tree shows the highest dissimilarity with the Bayesian posterior trees from which the Bayesian consensus tree is build (Fig.~\ref{Fig_Results-paircomp_within}-A). Note that it is also highly dissimilar to the Bootstraps. However that can be linked to the fact that both Bootstraps andÂ Bayesian posterior trees are measured as 125*1000 random pairwise comparisons instead of just 125 tree comparisons for the Bayesian consensus trees. The fact that the 1000 pairwise trees to be compared for each of the 125 comparisons are chosen randomly, it can add noise to the results for these two methods (see \hyperref[tree_comparisons]{methods above}). Also, in the case of the lower performances of the Bayesian consensus trees when using the Triplets distance, one can note the high overlap of the confidence intervals (Fig. ~\ref{Fig_Results-permeth_perparam} and Fig. ~\ref{Fig_Results-paircomp_within}-B).

% $Â§4: Thus, baycon seems the best way to go >...
% ^Â§5: >... but we have to take into account problems in our protocol.

%Â§5
%Caveats: the metrics
%   -RF is different and better than triplets.
\label{metrics_discussion}
This difference between the metrics can be explained by their respective ability to measure the differences between two trees \citep{kuhnerpractical2014}. As shown in Fig.~\ref{Fig_Results-permeth_perparam} and Fig.~\ref{Fig_Results-global_perparam}, the contrast between the results of the two metrics can be explained by the actual nature of both metrics (see \hyperref[SupplementaryMaterial]{Supplementary Material S2}):
\begin{enumerate}
\item{The Robinson-Foulds metric is a conservative metric, which is more sensible to single taxon displacement because it will count clades as similar only if they are composed of the same taxa with the same topologies \citep{RF1981}. When getting closer to the root of the tree, displacement of single taxon makes the clades not being exactly identical any more even if the clade still contains all the other taxa in both trees. This metric illustrates therefore the conservation of clades among the compared trees (i.e. when the normalised Robinson-Foulds distance is close to 1, most of the clades in the two trees are identical).}
\item{On the other hand, the Triplets method is measuring the position of each taxon towards two other reference taxa \citep{critchlowthe1996}. It will penalise only trees when taxa get removed furthest from their original position. This metric therefore illustrates the amount of wildcard taxa \citep{kearneyfragmentary2002} between the two compared trees (i.e. when the score is closed to 1, few wildcard taxa are present in the tree).}
\end{enumerate}
Therefore, in our study, both metrics are showing two different aspects of tree topology. For example, regarding the Bayesian consensus trees, completes clade are conserved most of the time (Fig.~\ref{Fig_Results-global_perparam}-top) but with an increasing amount of missing data, wildcard taxa get more and more displaced (Fig.~\ref{Fig_Results-global_perparam}-bottom). Because we are comparing the Baysian consensus tree to a single tree topology, the probability of recovering the exact position of a wildcard taxa than in the "best" tree is really low and even reducing with the amount of missing data. Regarding this fundamental differences, the Robinson-Foulds metric performs better in showing a distance from the true topology than the Triplets distance \citep{kuhnerpractical2014}. Therefore, the results for the Bayesian consensus trees can be preferred on the Maximum Likelihood trees because - when using the Robinson-Fould distance - they display a higher ability to recover the "correct" topology (Fig. ~\ref{Fig_Results-permeth_perparam}).

% $Â§5: The difference in the methods is not important if you look at Kuhner >...
% ^Â§6: >... but a final caveat is the way we simulate data.

%Â§6
%Caveats: the simulations
%   -Simulating morpho characters
The variations in our results could also be explained by another methodological caveat: the performance of our simulation protocol. In order to reduce the computational time of our analysis ($~$150 CPU years), we had to run our sensitivity analysis on modest size matrices (1000 molecular data and 100 morphological data). This matrix size is two orders of magnitude lower than some matrices used in published analysis \citep[e.g.][]{springermacroevolutionary2012,nithe2013} but still ranges in the sizes of smaller matrices used in other ones \citep[e.g.][]{sallam2011craniodental,kellymolecular2014}. %TG: note about NOT citing O'Leary here. 1-I don't like the paper ; 2-the Ni et al paper (primates) as a big morpho matrix too (1k  characters against 4k for O'Leary, same order of magnitude?) ; 3-We might want to limit the papers we're citing? note about citing Sean's and your paper here: 1-is it okay (regarding last time warning of being extra careful when citing you're own work)?
Also, simulating morphological characters is more complex than molecular ones. The underlying pattern of their evolution are often more complex and ruled by more parameters than molecular characters (i.e the number of character states, the states frequencies, the substitution matrix and the statistical model used) \citep{Pagel22011994,wagner2000,lewisa2001,wrightbayesian2014}. Morphological characters studies involve many potential statistical pitfalls (e.g. independent characters violation, rate variation - \citet{davalosintegrating2014}) and especially (i) incongruence with molecular signal and (ii) homoplasy \citep{wagner2000}.
\begin{enumerate}
\item{First, morphological data can display a different signal than molecular data, especially in small matrices \citep{wagner2000}. %REF
This might lead to a controversial phylogenetic signal in the overall matrix and lower down the support values. However, regarding empirical data studies, most of the groups shows fairly congruent morphological and molecular phylogenetic signal \citep[e.g.][]{leerates2013}.}
\item{Secondly, in this study, we made the assumption that theoretically, morphological characters are randomly distributed on an organism however it seems clear that empirical morphological data does not act randomly \citep{sansomfossilization2013,pattinsonphylogeny2014}. However, following our simulation assumption of random character distributions, if they accumulate through time in the same way as the majority of the molecular characters then, homoplasic characters are expected to appear randomly through time \citep{davalosintegrating2014}. Therefore, homoplasy is expected to be more important (by chance) in bigger morphological matrices \citep{davalosintegrating2014}. After reaching a critical amount of morphological characters, adding new ones increases homoplasy \citep{wagner2000}.}
\end{enumerate}

% $Â§6: Knowing the caveats, we can try to fix them >...
% ^Â§7: >... by taking into account data realness (fossil templates and rate heterogeneity).

%Â§7
%Further directions - is that necessary though?
%   -Using real data
One way to address this issue would be to run a similar analysis using real data. The amount of studies using Total Evidence data type matrices is steadily increasing \citep[e.g.][]{ronquista2012,slaterphylogenetic2013,beckancient2014} and one could analysis these total evidence matrices in the same framework as this study. However, the true topology (obtained from a matrix with no missing data) is unlikely to be known. A second way to address this issue, and especially to incorporate the variation of signal in morphological character would be to simulate morphological characters at different rates \citep{wrightbayesian2014} or to apply patterns of missing data directly obtained from the fossil record \citep{pattinsonphylogeny2014}. Nevertheless, our simulation already show clear results of the effect of missing data on topology in a Total Evidence framework. Also, our results show how recovering the correct topology can be improved by using Bayesian consensus trees in combination with a maximum amount of characters and a of living taxa with morphological data.
%   -Looking at branch length?

% $Â§7: But despite the caveats above, missing data is not a big deal >...
% ^Â§8: >... We can use TEM for integrating fossils into trees.

%Â§8
%Back to the importance of TEM - How we can use fossils (Intro-Â§2)
%   -Data is not a major problem
Despite the various caveats discussed above, we answered our main question on how does Total Evidence matrices perform in recovering topology with different methods and different amounts of missing data.
%   -Good and promising method.
Our results shows that this method performance decreases mainly with a reduction in the overlap of morphological data. In order to address this eventual problem, we propose to increase the data overlap by coding living species and to have a large morphological matrix. Also, our results shows disparity among methods and a clear superiority of the Bayesian consensus tree topology over the Maximum Likelihood one.

%---------------------------------------------
%
%       CONCLUSION
%
%---------------------------------------------

\section{Conclusion}

%Â§Conclusions
% part 1- Fossils are important and can be used combined with living taxa (Intro-Â§1 & Â§2)
%   -TEM works with missing data
% part 2- This is how we can improve the data to use them in the best way possible (Conclusion-Â§1 & Â§8)
%   -Use bayesian consensus
%   -Go code living taxa
Using Total Evidence matrices with missing data is not a problem for recovering the correct topology as long as enough morphological data is overlapping between the fossil and living taxa. In order to optimize good topological recovery, we propose to use the Bayesian consensus tree over the Maximum Likelihood tree or over a random subset of the Bayesian posterior trees distribution. Also, in order to improve the overlap in data in the morphological part of the matrix, we advice to use all the morphological characters available and to code them for a maximum of living taxa present in the matrix.


%---------------------------------------------
%
%       ...
%
%---------------------------------------------
\section{Supplementary material}
Supplementary material (code, analysis and full results) can be found in the Dryad data repository at http://dx.doi.org/10.5061/dryad.XXXX. %section and sentence format asked by Syst. Biol.

\section{Acknowledgements}
Thanks to Fr\'{e}d\'{e}ric Delsuc, Emmanuel Douzery, Andrew Jackson, Gavin Thomas and April Wright. %TG : Not sure how many people we have to thank but it could also include Graham Slater, Nick Matzke and Trevor Hodkinson.
for useful comments on our simulation protocol.
%Thanks to Sive Finlay, Kevin Healy and Adam Kane for comments on the manuscript. %TG: or helping writing sentences does not really count.
Thanks to Paddy Doyle, Graziano D'Innocenzo and Sean McGrath for assistance with the computer cluster. Simulations used the Lonsdale cluster maintained by the Trinity Centre for High Performance Computing and funded through grants from Science Foundation Ireland. This work was funded by a European Commission CORDIS Seventh Framework Programme (FP7) Marie Curie CIG grant (proposal number: 321696).

%cite cheat sheet
 % The \cite command functions as follows:
 %   \citet{key} ==>>                Jones et al. (1990)
 %   \citet*{key} ==>>               Jones, Baker, and Smith (1990)
 %   \citep{key} ==>>                (Jones et al., 1990)
 %   \citep*{key} ==>>               (Jones, Baker, and Smith, 1990)
 %   \citep[chap. 2]{key} ==>>       (Jones et al., 1990, chap. 2)
 %   \citep[e.g.][]{key} ==>>        (e.g. Jones et al., 1990)
 %   \citep[e.g.][p. 32]{key} ==>>   (e.g. Jones et al., p. 32)
 %   \citeauthor{key} ==>>           Jones et al.
 %   \citeauthor*{key} ==>>          Jones, Baker, and Smith
 %   \citeyear{key} ==>>             1990

\bibliographystyle{sysbio}
\bibliography{References}
%Needs to be cleaned a bit

%---------------------------------------------
%
%       Supplementary, separated
%
%---------------------------------------------

%Ignore this for 'fast' pdf building
%\section{Supplementary Material}

%    \input{SupplementaryMaterial/SupplementaryMaterial}
%    \label{SupplementaryMaterial}

%END
\end{document}